src/pybind/cephfs/cephfs.pyx

```diff
@@ -481,6 +481,17 @@ cdef class LibCephFS(object):
             for key, value in conf.items():
                 self.conf_set(key, value)
 
+    def get_fscid(self):
+        """
+        Return the file system id for this fs client.
+        """
+        self.require_state("mounted")
+        with nogil:
+            ret = ceph_get_fs_cid(self.cluster)
+            if ret < 0:
+                raise make_ex(ret, "error fetching fscid")
```

Comment:
```suggestion
        if ret < 0:
            raise make_ex(ret, "error fetching fscid")
```

to fix

https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=centos8,DIST=centos8,MACHINE_SIZE=gigantic/54571//consoleFull

Reply:
thx. done!